guides/source/command_line.md

```diff
@@ -395,10 +395,10 @@ INFO: You can also use `rake -T`  to get the list of tasks.
 ```bash
 $ bin/rake about
 About your application's environment
-Rails version             4.2.0
-Ruby version              1.9.3 (x86_64-linux)
-RubyGems version          1.3.6
-Rack version              1.3
+Rails version             5.0.0
+Ruby version              2.2.0 (x86_64-linux)
+RubyGems version          2.4.5
+Rack version              1.5
```

Comment:
Rails is using [rack 1.6](https://github.com/rails/rails/blob/master/actionpack/actionpack.gemspec#L24), this should probably reflect that.